docs/user/commercial/index.rst

```diff
@@ -1,10 +1,41 @@
-Read the Docs for Business
---------------------------
+About |com_brand|
+=================
```

Comment:
Does this work? I never know if it's going to expand first or not.

Reply:
This one worked, but we can't write `*|com_brand|*` :facepalm: 

![image](https://user-images.githubusercontent.com/374612/218106177-2bc8bada-5cd9-432b-a0d5-41c088f98f3a.png)

Reply:
Yea... could be a good use case for myst, but not sure if they have substitutions heh :)